.github/workflows/build-and-publish-proto-bindings.yml

```diff
@@ -0,0 +1,53 @@
+name: Protobuf Bindings
+
+on: push
+#  pull_request:
+#  push:
+#    branches:
+#      - main
```

Comment:
Does this need changing before merging?

Also line 35 below:
```yml
        # if: github.event_name == 'release'
```

We don't want every push to publish a new copy of this stuff.

Reply:
Yes. I will need to change this. I was using `push` to test nexus publishing. 

I'll clean up line 35. We don't need it.